everviz/plugins/_lineplot_plugin/lineplot_dropdown_plugin.py

```diff
@@ -0,0 +1,85 @@
+import dash_html_components as html
+from uuid import uuid4
+
+from webviz_config import WebvizPluginABC
+
+from everviz.data.load_csv._get_data import get_data
+from everviz.plugins._lineplot_plugin._dash_components._lineplot_callback import (
+    _lineplot_callback,
+)
+from everviz.plugins._lineplot_plugin._dash_components._lineplot_layout import (
+    _get_layout,
+)
+from dash.dependencies import Input, Output
+import plotly.graph_objs as go
+import dash_core_components as dcc
+
+
+class LineplotDropdownPlugin(WebvizPluginABC):
```

Comment:
There are no tests for the LineplotDropdownPlugin, not of the layouts or the callbacks either